blitz.config.ts

```diff
@@ -3,7 +3,7 @@ import { sessionMiddleware, simpleRolesIsAuthorized } from "blitz"
 module.exports = {
   middleware: [
     sessionMiddleware({
-      cookiePrefix: 'CodersCrewApp',
+      cookiePrefix: "CodersCrewApp",
```

Comment:
Why we have double quotes here when we set Prettier to use single ones?

Reply:
Now I think it's better to use double quotes because in majority of file is used double quotes by default 😂